.github/workflows/ci.yml

```diff
@@ -161,38 +161,48 @@ jobs:
     needs: pre-commit
     uses: ./.github/workflows/python-package-create.yml
     with:
-      repository: beeware/${{ matrix.repo }}
-      tox-source: ${{ matrix.tox-source }}
-      tox-factors: ${{ matrix.tox-factors }}
-      distribution-path: ${{ matrix.dist-path }}
+      repository: beeware/${{ matrix.repo || matrix.name }}
       build-subdirectory: ${{ matrix.build-subdir }}
     strategy:
       fail-fast: false
       matrix:
-        repo: [ briefcase, gbulb, rubicon-objc, toga ]
+        name:
+        - beeware
+        - briefcase-automation
+        - gbulb
+        - rubicon-objc
+        - toga-core
+        - toga-android
+        - toga-chart
+        - travertino
         include:
-        - tox-source: .[dev]
-        - dist-path: dist
-        - repo: briefcase
-          tox-source: .[dev]
-          tox-factors: -with-automation
-          dist-path: dist
-        - repo: gbulb
-          tox-source: tox
-          dist-path: dist
-        - repo: toga
-          tox-source: ./core[dev]
-          dist-path: "*/dist/*"
+        - name: briefcase-automation
+          repo: briefcase
+          build-subdir: automation
+        - name: toga-core
+          repo: toga
           build-subdir: core
+        - name: toga-android
+          repo: toga
+          build-subdir: android
+
+  test-package-briefcase:
```

Comment:
I had to separate building the Briefcase package out because the `artifact-name` output from the `python-package-create.yml` becomes unstable when it is used in a matrix. This is under-designed in the GitHub Actions system and it just takes on the value from whichever job set it last. I found several proposals to fix this but GitHub hasn't gotten around to this. There are a few third-party workarounds to this but they seem very heavy-handed for our use-case.

Furthermore, my whole goal with outputting the Artifact's name was to abstract the packaging process from its downstream use; for the most part, I've achieved that. However, parts of the implementation are assuming that all related packages share a common base name. So, for Toga, it's assumed all of the Artifact's names start with `Packages-toga`....a more complicated implementation could probably workaround this but whatever....